.github/workflows/update-lts-tags.yaml

```diff
@@ -39,12 +39,12 @@ jobs:
         run: |
           CHANGES=$(git status --porcelain=v1 2>/dev/null | wc -l)
           if [ ${CHANGES} -ne 0 ]; then
-            gh pr list --state open | grep "Update appVersions for DC apps"
-            if [ $? -ne 0 ]; then
+            OPEN_PRS=$(gh pr list --state open | grep "Update appVersions for DC apps" || true)
+            if [[ ${OPEN_PRS} ]]; then
```

Comment:
and does this change introduce any new functionality? I see this as a refectory for the existing code, right?

Reply:
No new functionality. It's just not to fail the script if no existing update appVersion PRs are found. Previously it was relying on exit code of the command, which did not work well.

https://github.com/atlassian/data-center-helm-charts/pull/478 is raised by this workflow